specification/liftrqumulo/resource-manager/Qumulo.Storage/stable/2024-06-19/Qumulo.Storage.json

```diff
@@ -0,0 +1,712 @@
+      "get": {
+        "operationId": "FileSystems_ListByResourceGroup",
+        "tags": [
+          "FileSystems"
+        ],
+        "description": "List FileSystemResource resources by resource group",
+        "parameters": [
+          {
+            "$ref": "../../../../../common-types/resource-management/v3/types.json#/parameters/ApiVersionParameter"
+          },
+          {
+            "$ref": "../../../../../common-types/resource-management/v3/types.json#/parameters/SubscriptionIdParameter"
+          },
+          {
+            "$ref": "../../../../../common-types/resource-management/v3/types.json#/parameters/ResourceGroupNameParameter"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "Azure operation completed successfully.",
+            "schema": {
+              "$ref": "#/definitions/FileSystemResourceListResult"
+            }
+          },
+          "default": {
+            "description": "An unexpected error response.",
+            "schema": {
+              "$ref": "../../../../../common-types/resource-management/v3/types.json#/definitions/ErrorResponse"
+            }
+          }
+        },
+        "x-ms-examples": {
+          "FileSystems_ListByResourceGroup": {
+            "$ref": "./examples/FileSystems_ListByResourceGroup_MaximumSet_Gen.json"
+          },
+          "FileSystems_ListByResourceGroup_MinimumSet_Gen": {
+            "$ref": "./examples/FileSystems_ListByResourceGroup_MinimumSet_Gen.json"
+          }
+        },
+        "x-ms-pageable": {
+          "nextLinkName": "nextLink"
+        }
+      }
+    },
+    "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Qumulo.Storage/fileSystems/{fileSystemName}": {
+      "get": {
+        "operationId": "FileSystems_Get",
+        "tags": [
+          "FileSystems"
+        ],
+        "description": "Get a FileSystemResource",
+        "parameters": [
+          {
+            "$ref": "../../../../../common-types/resource-management/v3/types.json#/parameters/ApiVersionParameter"
+          },
+          {
+            "$ref": "../../../../../common-types/resource-management/v3/types.json#/parameters/SubscriptionIdParameter"
+          },
+          {
+            "$ref": "../../../../../common-types/resource-management/v3/types.json#/parameters/ResourceGroupNameParameter"
+          },
+          {
+            "name": "fileSystemName",
+            "in": "path",
+            "description": "Name of the File System resource",
+            "required": true,
+            "type": "string",
+            "pattern": "^[a-zA-Z0-9_-]*$"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "Azure operation completed successfully.",
+            "schema": {
+              "$ref": "#/definitions/LiftrBase.Storage.FileSystemResource"
```

Comment:
Is there a way to get rid of the prefix here LiftrBase.Storage ? it's causing JS sdk breaking.

Reply:
we are fine with completely deprecating previous version and just use the new one since there are no real production customer user

Reply:
For Swagger API, we anyway have the Breaking change approval